src/quick-tour/checkout-page.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Checkout Page
+---
+
+### The checkout consists of two steps:
+
+1. Shipping Information
+
+   The first step of the checkout process is for the customer to complete the shipping address information, and to choose the shipping method. If the customer has an account, the shipping address is entered automatically, but can be changed if needed.
+
+   ![Example storefront checkout page]({% link images/images/storefront-checkout-shipping-full.png %}){: .zoom}
+   _Checkout Page (Shipping)_
+
+1. Review and Payment Information
+
+   The second step of the checkout process is for the customer choose the Payment method and Apply the Discount code and get good discount.
+
+   ![Example storefront checkout page]({% link images/images/storefront-checkout-payment-full.png %}){: .zoom}
+   _Checkout Page (Review & Payments)_
+
+The progress bar at the top of the page follows each step of the checkout process, and the Order Summary shows the information entered so far.
+
+The only exception is checkout of virtual and/or downloadable products: if there are only these types of products in the shopping cart, checkout is automatically transformed to one-step procedure, because shipping information is not required.
```

Comment:
```suggestion
{:.bs-callout-info}
The exception to a two-step checkout applies to virtual and/or downloadable products---if there are only these types of products in the shopping cart, checkout is automatically transformed to one-step procedure, because shipping information is not required.
```